app/generated_resources.grd

```diff
@@ -6851,6 +6851,9 @@ Keep your key file in a safe place. You will need it to create new versions of y
       <message name="IDS_TAB_SEARCH_RECENTLY_CLOSED_TABS" desc="The label on the tab search list section corresponding to recently closed tabs.">
         Recently Closed Tabs
       </message>
+      <message name="IDS_SETTINGS_TABS_SEARCH_SHOW" desc="The button for searching tabs on the left of tab strip region.">
+        Show Search Tabs Button
+      </message>
```

Comment:
This should go into `app/brave_generated_resources.grd`. `app/generated_resources.grd` gets automatically regenerated with each Chromium bump from the upstream source.